### ISSUE TITLE
NIC ice layers

### ISSUE THREAD

**Issue by kbailey-noaa** [asascience-open/IOOS-EDS#210]
For discussion:
User request (Bryan Basher) from the National Ice Center.  2 sources for integration: 

1. [Interactive Multisensor Snow and Ice Mapping System (IMS)](https://www.natice.noaa.gov/ims/index.html) provides 1 km resolution of snow and ice in the Northern Hemisphere.
Digging a little, that page leads to: https://www.natice.noaa.gov/ims/grib2_v3_p1.html -- but notice they organize files by year, so the URL will change every January 1.  Should be the 1 km version.
2. [Great Lakes sea ice concentration](https://www.natice.noaa.gov/products/grib.html).

If the upgraded EDS ingest system makes source integration faster, then I'd definitely like to pursue this for FY20, assuming level of effort is reasonable/low and we don't burn the budget.   TBD, but I'm tentatively adding this to the milestone.  I realize this is scope creep, but looking at the issues under the FY20 Source Integration milestone, we could push CDIP nowcasts out if necessary.  I'm more interested in creating/maintaining these relationships with users (NIC and CO-OPS).

Also, this is related to issue #196 - Add OFS ice variables.

**Comment by kbailey-noaa** [comment]
@Bobfrat @kknee 
Do you have an update on this request? The NIC POC sent an email that I didn't respond to.   I'd like to try to provide them an idea of what we need to do so they can understand level of effort, whether or not they can provide us with anything additional, and a general timeline for when they might be able to see this.

**Comment by Bobfrat** [comment]
@kbailey-noaa we took a look at this source and it should be a reasonably low level of effort to get into the new data ingest system. I see this milestone has a due date of March 31 but a tag of priority. Does that mean you'd like this integrated ASAP? 

What we'll need from NIC is probably just metadata. We can put together a list of the attributes we'll need from them.

**Comment by kbailey-noaa** [comment]
@Bobfrat Yeah, ASAP.  I had written FY20 in the issue but I should've been clearer about timing. My perception was that items in this milestone are rolled out as ready with a final deadline of 3/31, rather than wait on everything until 3/31.  
Since this is a customer request I want to be able to show we're responsive and willing to work with them, as we continue promoting the THREDDS and Model Viewer.  So user requests tend to be high priority (within reason).
I'll respond to NIC to just indicate we're working on this and will reach out with metadata questions when ready. Thanks!

**Comment by kbailey-noaa** [comment]
@Bobfrat status?

**Comment by kknee** [comment]
@bobfrat - I just enabled the Great Lakes NIC layer in dev, let's take a quick look tomorrow.
